functions/workershops/san-jose.js

```diff
@@ -0,0 +1,7 @@
+export async function onRequest() {
+  const url = new URL(
+    "https://www.ortusclub.com/cloudflare-developers-workershop-san-jose"
```

Comment:
would like to parameterize this better in the future, but this is a fine first pass